documentation/modules/con-kafka-connect-logging.adoc

```diff
@@ -0,0 +1,54 @@
+// This assembly is included in the following assemblies:
+//
+// assembly-deployment-configuration-kafka-connect.adoc
+
+[id='con-kafka-connect-logging-{context}']
+= Kafka Connect loggers
+
+Kafka Connect has its own configurable loggers:
+
+* `connect.root.logger.level`
+* `log4j.logger.org.apache.zookeeper`
```

Comment:
Connect doesn't connect to Zookeeper. Are you sure it has `log4j.logger.org.apache.zookeeper` and `log4j.logger.org.I0Itec.zkclient` loggers? Or is this just some bug copied from previous version of the doc?

Reply:
Certainly copied from the previous doc. I see these loggers described in relation to Kafka Connect: https://fossies.org/linux/kafka/config/connect-log4j.properties

Reply:
@tombentley I get it that reflections logger might be used. But do you have any idea why would Conenct use Zookeeper and zkClient loggers? I think this must be a mistake in the Kafka files, or?

Reply:
After a bit of digging... the zookeeper lines in the connect log4j seem to date back to 2015. It seems likely to me that at that time Connect did connect to ZK, but it doesn't do so any more. So I think we should remove this from the example.

Reply:
@scholzj @tombentley - Thanks. I have removed `log4j.logger.org.apache.zookeeper` and `log4j.logger.org.I0Itec.zkclient` loggers from the lists of Kafka Connect loggers and Kafka Connect with S2I loggers